tex/gregoriotex-spaces.tex

```diff
@@ -1590,6 +1590,9 @@
 % #1 : line number
 % #2-#4 : same as #1-#3 of \grechangedim
 \def\grechangenextscorelinedim#1#2#3#4{%
+  \IfStrEqCase{#2}{{abovelinestextheight}{}{noteadditionalspacelinestext}{}{spaceabovelines}{}{abovelinestextraise}{}{spacelinestext}{}{spacebeneathtext}{}{translationheight}{}}[%
```

Comment:
Just to make sure I'm reading this right, you're using `\IfStrEqCase` to construct a `not (... and ... and ...)` sequence that will raise an error if this macro is called with an unsupported distance.

Reply:
That's right.

Reply:
Oh, actually, it's `if not (... or ... or ...) then print error`.

Reply:
Yeah, that's what I meant.